resolwe_bio/tests/processes/test_import.py

```diff
@@ -156,14 +157,52 @@ def external_test_sra(self):
 
     @tag_process("basespace-file-import")
     def external_test_basespace_import(self):
+        output_folder = Path("basespace_import") / "output"
         # Token with limited scope pre-obtained from dedicated BaseSpace testing app.
         handle = Secret.objects.create_secret(
-            "9bdf059c759a429f8af52ca084130060", self.admin
+            "d23ee29d4db4454480ad89d925475913", self.admin
         )
 
-        inputs = {"file_id": "9461130722", "access_token_secret": {"handle": handle}}
+        inputs = {"file_id": "25157957505", "access_token_secret": {"handle": handle}}
         file = self.run_process("basespace-file-import", inputs)
 
         self.assertFile(
-            file, "file", "Test_S1_L001_R1_001.fastq.gz", compression="gzip"
+            file,
+            "file",
+            output_folder / "single_S1_L001_R1_001.fastq.gz",
+            compression="gzip",
+        )
+
+        inputs = {
+            "file_id": "25157957505",
+            "access_token_secret": {"handle": handle},
+            "advanced": {"output": "filename", "tries": 6, "verbose": True},
+        }
+        file = self.run_process("basespace-file-import", inputs)
+
+        self.assertFile(
+            file,
+            "file",
+            output_folder / "single_S1_L001_R1_001.fastq.gz",
```

Comment:
My IDE says `fn` is expecting `str` but got `Path`. Wrapping the line into

```python
str(output_folder / "single_S1_L001_R1_001.fastq.gz")
```
makes it stop complaining.